CODEOWNERS

```diff
@@ -101,6 +101,7 @@ tests/scripts/run_performance.sh @tt-rkim @tapspatel
 tt_eager/tt_dnn/op_library/bmm/ @TT-BrianLiu @bbradelTT
 
 # eager - tensor and op infra
+tt_eager/tt_dnn/op_library/ccl/* @SeanNijjar
```

Comment:
Delete the `*` - I think github may only match the first level. Need to confirm that though...